server/utils/utils.ts

```diff
@@ -14,4 +16,9 @@ const properCaseName = (name: string): string => (isBlank(name) ? '' : name.spli
 const convertToTitleCase = (sentence: string): string =>
   isBlank(sentence) ? '' : sentence.split(' ').map(properCaseName).join(' ')
 
+export const addSelect = (selectItems: SelectOption[], text = 'Select'): SelectOption[] => {
+  selectItems.unshift({ value: '', text })
```

Comment:
Does it make sense to `[{ value: '', text }, ...selectItems]` instead to prevent mutating the input?

Reply:
Yeah definitely.  Good spot.